### DISCUSSION TITLE
请问，eggjs有限制同一IP在一定时间段请求次数的方法吗？

### DISCUSSION THREAD

**Discussion by 983796605** [eggjs/egg#4995] [Q&A] [answered]
请问，eggjs有限制同一IP在一定时间段请求次数的方法吗？

**Comment by huihuicn** [comment]
这个要自己写，或者放到nginx里面

**Comment by JimmyDaddy** [comment, marked as answer]
可以结合 redis 自己写一个，或者用这个 [ratelimit](https://github.com/koajs/ratelimit)

或者参考这个：https://github.com/eggjs/egg/issues/3382
